examples/emberjs/todomvc/config/environment.js

```diff
@@ -10,10 +10,9 @@ module.exports = function (environment) {
 		 *   locationType: 'history'
 		 *
 		 * These settings are required by todomvc's (the repo) tests
+		 * (And the deployed site -- using GH Pages to point at the pre-built assets in source)
 		 */
-		// rootURL: '',
-		// locationType: 'none',
-		rootURL: '/',
+		rootURL: '/examples/ember/dist/',
```

Comment:
should this point to the 'emberjs' folder instead of 'ember' ?

Reply:
I would probably only change the rootURL for production, since you want to be able to work locally in the emberjs/todomvc directory and spin up the app directly. 

How about this:
```
if (environment === 'production') {
		// here you can enable a production-specific feature
    ENV.rootURL = '/examples/emberjs/todomvc/dist/';
	}
```

Reply:
the URL prefix is still usable in development locally